app/src/main/java/org/p2p/wallet/striga/wallet/models/StrigaOnchainTxType.kt

```diff
@@ -0,0 +1,17 @@
+package org.p2p.wallet.striga.wallet.models
+
+enum class StrigaOnchainTxType {
```

Comment:
I've already added `status` and `StrigaOnchainTxStatus` and they're actually don't represent actual tx status, but StrigaOnchainTxType represents status but not a type. Naming is a weak side of Striga. 
I think it's better to keep original names to avoid confusion until we start using all this data.